docs/tutorials/requests.md

```diff
@@ -0,0 +1,123 @@
+# Calling an API using Requests
+
+This tutorial will show you how to interact with an API using the [Requests](https://requests.readthedocs.io/en/master/) library. Requests is a popular library, but it doesn't work out of the box with Pyscript. We will use the [pyodide-http](https://github.com/koenvo/pyodide-http) library to patch the Requests library, so it works with Pyscript.
```

Comment:
it currently doesn't work out of the box with Pyscript because of limitations related to webworkers.

Maybe we should explain that's not a long lasting feature somehow? Not sure.

Reply:
I think the issue with the requests library is that it depends on sockets and sockets arent supported in pyodide yet, is this right or am I confusing with something? 🤔

Reply:
I think Fabio is correct here - some [discussion from the Pyodide team](https://github.com/pyodide/pyodide/issues/398) about `requests`, blocking, and the socket layer.

Reply:
Aha! I thought it didn't because of blocking calls or smth 😅
Okay! Sorry :) but yeah your choice to add it or not I guess, also for tests. LGTM